Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -396,11 +397,25 @@ function Invoke-CodeGen {
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
         $tool = (Join-Path $toolsPath 'EdFi.Ods.CodeGen')
         $repositoryRoot = (Get-RepositoryRoot $implementationRepo).Replace($implementationRepo, '')
-
-        if ($IncludePlugins) {
+              
+        if (-not [string]::IsNullOrWhiteSpace($ExtensionPaths)) {           
+            Foreach ($extensionPath in $ExtensionPaths) {        
+                if (![System.IO.Directory]::Exists($extensionPath)) {
+                    throw "Unable to find extension Location project path at location $extensionPath ."
+                }
+            }
+        }
```

Comment:
this looks like it's similar code from codegen. should we let the console app handle the errors?

Reply:
yes

Reply:
As discussed ,  I removed this condition as I did in codegen C# code